### ISSUE TITLE
Cluster info not working for me

### ISSUE THREAD

**Issue by Letoric** [Qowyn/ark-tools#10]
Hello Qowyn. I'm not able to get the cluster information to update for me. Can you guide me on what I'm doing wrong by chance? It's coming back with a 'potentially corrupt character data' error. I deleted the character file, and then re-uploaded an item, and it came back with the same error.

**Comment by TheRealestMVP** [comment]
Same here.

```
Found potentially corrupt cluster data: /path/to/cluster/data/<steam-id>
```

This is printed for all Steam IDs in the cluster directory. Nothing is written to the given output directory.

The cluster is working like a charm, also ark-tools for extracting all other information from the save games. Only extracting the cluster data fails.

**Comment by Qowyn** [comment]
Thanks for the repots, this is now fixed with version 0.5.1
